packages/backend/package.json

```diff
@@ -113,7 +113,7 @@
     "esbuild": "^0.15.12",
     "esbuild-register": "^3.3.3",
     "eslint-config-custom": "*",
-    "miniflare": "^2.12.1",
+    "miniflare": "^3.20240304.0",
```

Comment:
Is there a possibility that this one will introduce breaking changes?

Reply:
This is a dev dependency only used in our backend tests. I would not consider it a breaking change. Do you have something in mind that could be broken by this?